server/status/runtime.go

```diff
@@ -170,6 +179,18 @@ func (rsr *RuntimeStatSampler) SampleEnvironment() {
 		log.Errorf(context.TODO(), "unable to get cpu usage: %v", err)
 	}
 
+	fds := gosigar.ProcFDUsage{}
+	if err := fds.Get(pid); err != nil {
```

Comment:
Ack. Should have looked at that closer. The lack of comments in gosigar is nice.